src/Symfony/Component/Workflow/SupportStrategy/SupportStrategyInterface.php

```diff
@@ -11,10 +11,14 @@
 
 namespace Symfony\Component\Workflow\SupportStrategy;
 
+@trigger_error(sprintf('"%s" is deprecated since Symfony 4.1. Use "%s" instead.', SupportStrategyInterface::class, WorkflowSupportStrategyInterface::class), E_USER_DEPRECATED);
+
 use Symfony\Component\Workflow\Workflow;
 
 /**
  * @author Andreas Kleemann <akleemann@inviqa.com>
+ *
+ * @deprecated
```

Comment:
`@deprecated since version 4.1, to be removed in 5.0. Use WorkflowSupportStrategyInterface instead`